src/app_service/service/transaction/service.nim

```diff
@@ -474,27 +472,22 @@ QtObject:
         finalPassword = common_utils.hashPassword(password)
     try:
       var chainID = 0
-      var isEthTx = false
 
       if(selectedRoutes.len > 0):
         chainID = selectedRoutes[0].fromNetwork.chainID
 
-      var tokenSymbol = ""
-      if self.isCollectiblesTransfer(sendType):
-        tokenSymbol = assetKey
-      else:
-        let token = self.tokenService.getTokenBySymbolByTokensKey(assetKey)
-        if token != nil:
-          tokenSymbol = token.symbol
-
+      let token = self.tokenService.getTokenBySymbolByTokensKey(assetKey)
```

Comment:
hmm doesn't seem right to be calling this for a non ETH/ERC20 transfer... 
It seems to me that, with these changes, `transferToken` will misbehave for a collectibles transfer if `token` is non-nil. Are we 100% sure we cannot craft some ERC20 token and some ERC721 token for which that condition will be triggered?

Reply:
Thanks for the heads up, I thought I tested with community collectibles but now I realized they must have been assets. I see the issue and will correct it to use only assetKey to extract the token identity.